RevenueCatUI/CustomerCenter/CustomerInfo+CurrentEntitlement.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfo+CurrentEntitlement.swift
+//
+//  Created by Cesar de la Vega on 17/7/24.
+
+import Foundation
+import RevenueCat
+
+extension CustomerInfo {
+
+    /// Returns the earliest expiring iOS App Store entitlement. If this CustomerInfo contains multiple lifetime 
+    /// entitlements and no expiring entitlements, the returned entitlement is undefined.
+    func earliestExpiringAppStoreEntitlement() -> EntitlementInfo? {
+        return self.entitlements
+            .active
+            .values
+            .lazy
+            .filter { $0.store == .appStore }
```

Comment:
[not for this PR] I was thinking, if the user has a purchase in a different store, we could probably still get it and display whatever information is available in the "no_active" page I guess but nothing to do here right now.

Reply:
good point, will take notes of that